### PR TITLE
cato-networks-c2c-docs

### PR DESCRIPTION
**Pull request by JV0812**
- Created C2C doc for Cato Networks
- CID added
- Card added in index page

## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1204369149242983

### CODE REVIEW

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/cato-networks-source.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+## Data Sources
+
+Cato Networks source ingests security events from [EventsFeed API](https://support.catonetworks.com/hc/en-us/articles/360019839477-Cato-API-EventsFeed-Large-Scale-Event-Monitoring) and audit events from [AuditFeed API](https://support.catonetworks.com/hc/en-us/articles/360017900857-Cato-API-AuditFeed).
+
+## Prerequisite
+
+1. Only account administrators with the Editor privilege can generate keys in the Cato Management Application (CMA).
+1. For ingesting security events you need to enable the events feeds in your account. Follow the below-mentioned steps to enable events feed:
+    - In the navigation panel, select **System > API Access Management**.
+    - Select **Event Feed Enabled**. <br/>After this, your account starts sending events to the Cato API server. <br/><img src={useBaseUrl('img/send-data/cato-c2c/cato-networks-enable-events-feed.png')} alt="cato-networks-enable-events-feed.png" width="700" />
```
Comment (JainM6) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/cato-networks-source.md:25]:
```suggestion
    - Select **Event Feed Enabled**. After this, your account starts sending events to the Cato API server. <br/><img src={useBaseUrl('img/send-data/cato-c2c/cato-networks-enable-events-feed.png')} alt="cato-networks-enable-events-feed.png" width="700" />
```

Comment (JainM6) [reply docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/cato-networks-source.md:25]: All C2Cs images are saved under the /img/send-data/. No need of separate folders.

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/cato-networks-source.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+### API Key
+
+All access to Cato networks require an API Key. Follow the below instructions to set up an API Key.
+
+1. In the navigation menu, click **Administration > API Management**. <br/><img src={useBaseUrl('img/send-data/cato-c2c/cato-networks-administration.png')} alt="cato-networks-administration.png" width="700" />
+1. On the **API Keys** tab, click **New**. The **Create API Key** panel opens. 
+1. Enter a **Key Name**. <br/><img src={useBaseUrl('img/send-data/cato-c2c/cato-networks-new-api-key.png')} alt="cato-networks-new-api-key.png" width="250" />
+1. Select **View** in the **API Permission**.
+1. Select **Any IP** to allow this API key for any IP address under the **Allow Access from IPs** section.
+1. (Optional) Select a date that the API key Expires at. If you select an expiration date then you need to update the source configuration with a new API Key or else an unauthorized error will be received.
```
Comment (JainM6) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/cato-networks-source.md:40]:
```suggestion
1. (Optional) Select a date when the API key expires. If you select an expiration date, then you need to update the source configuration with a new API key, or else an unauthorized error will be received.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/cato-networks-source.md [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+1. Click **Apply**. The API key is added and a popup window containing the new API key is displayed.
```
Comment (JainM6) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/cato-networks-source.md:41]:
```suggestion
1. Click **Apply**. The API key is added, and a popup window containing the new API key is displayed.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/cato-networks-source.md [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+1. Click **Copy** to copy the API Key that is generated by the Cato Management Application and save it to a secure location.
+:::note
+You cannot access the value for the API key once you close this window.
```
Comment (JainM6) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/cato-networks-source.md:44]:
```suggestion
Note that the API key value will not be available after closing this window. Kindly ensure that you copy and securely save the API key before closing the window.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/cato-networks-source.md [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+If the Source has any issues during any one of these states, it is placed in an **Error** state.
+
+When you delete the Source, it is placed in a **Stopping** state. When it has successfully stopped, it is deleted from your Hosted Collector.
+On the [Collection page](/docs/manage/health-events#collection-page), the Health and Status for Sources is displayed. You can click the text in the Health column, such as **Error**, to open the issue in Health Events to investigate.
+
+## Setup Cato Networks Source
+
+When you create an Cato Networks Source, you add it to a Hosted Collector. Before creating the Source, identify the Hosted Collector you want to use or create a new Hosted Collector. For instructions, see [Configure a Hosted Collector](/docs/send-data/hosted-collectors/configure-hosted-collector).
+
+To configure an Cato Networks Source:
```
Comment (JainM6) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/cato-networks-source.md:71]:
```suggestion
To configure a Cato Networks Source:
```

**Review by JainM6 — CHANGES_REQUESTED** [review]
Some formatting changes.

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/cato-networks-source.md [hunk 6]
```diff
@@ -0,0 +1,151 @@
...
+
+This source ingests Security and Audit events from Cato API.
+
+## Data Sources
+
+Cato Networks source ingests security events from [EventsFeed API](https://support.catonetworks.com/hc/en-us/articles/360019839477-Cato-API-EventsFeed-Large-Scale-Event-Monitoring) and audit events from [AuditFeed API](https://support.catonetworks.com/hc/en-us/articles/360017900857-Cato-API-AuditFeed).
+
+## Prerequisite
+
+1. Only account administrators with the Editor privilege can generate keys in the Cato Management Application (CMA).
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/cato-networks-source.md:22]: @JV0812 as we discussed, these should be bulleted items (not numbered steps).

**Review by kimsauce — DISMISSED** [review]
Please apply mine & Megha's edits, then click "Re-request review".

**Review by JainM6 — APPROVED** [review]
Thank you. Looks good to me.

**Pull request merged**